### ISSUE TITLE
Feature request: le9 patchset support

### ISSUE THREAD

**Issue by IronOverheat** [Frogging-Family/linux-tkg#283]
le9 is kernel patch already picked up by XanMod that allows for "protecting the file cache from being evicted from RAM [...] protects clean file pages under memory pressure to prevent thrashing and what users normally encounter with high latency and locking issues in near out-of-memory conditions"
"**According to the Phoronix reader involved with the Le9 work, he's reportedly able to run Mozilla Firefox with 37 tabs as well as having Skype, Discord, two PDFs, and LibreOffice all running on an aging decade old system with just 2GB of RAM**"
https://www.phoronix.com/scan.php?page=news_item&px=le9-Linux-Low-RAM
https://github.com/hakavlad/le9-patch

**Comment by hakavlad** [comment]
>le9 patchset

It's not a patchset, it's a single simple patch.

**Comment by ptr1337** [comment]
Really?  Thanks for that comment. but that it is providing many help on lower systems, also on higher systems when compiling, its a good patch. 

There is also for it.

**Comment by travankor** [comment]
What about mglru patchset? It's available for xanmod and liquorix/zen kernel. The Multigen LRU framework addresses issues with the current kernel design, "the current page reclaim is too expensive in terms of CPU usage and often making poor choices about what to evict. We would like to offer an alternative framework that is performant, versatile and straightforward."

https://www.phoronix.com/scan.php?page=news_item&px=Multigen-LRU-Framework-V3
https://lwn.net/Articles/856931/

**Comment by Tk-Glitch** [comment]
@travankor We are already offering that patchset in the community patches repo

**Comment by ryanmusante** [comment]
Is this patchset still in community repo? I don't see it.

**Comment by AdelKS** [comment]
@ryanmusante yes: https://github.com/Frogging-Family/community-patches/tree/master/linux516-tkg
